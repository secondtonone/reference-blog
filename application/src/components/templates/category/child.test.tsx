/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/react';
import SubcategoryPage from './child';

import fixtureArticles from '~/__fixtures__/articles';
import fixtureCategories from '~/__fixtures__/categories';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      asPath: '/blog/',
      query: {
        page: 1
      }
    };
  },
}));

describe('Match snapshots SubcategoryPage', () => {
  it('default', () => {
    const pageProps = {
      categoryFields: {
        ...fixtureCategories[1],
        parent: {
          ...fixtureCategories[0],
          children: [
            fixtureCategories[2],
            fixtureCategories[3]
          ]
        }
      },
      articles: fixtureArticles,
      breakpoint: { isPhone: false, isTablet: false, isDesktop: true },
      subcategory: fixtureCategories[1].slug
    };

    const { asFragment } = render(
      <SubcategoryPage {...pageProps} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
