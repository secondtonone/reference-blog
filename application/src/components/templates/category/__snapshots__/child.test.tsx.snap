// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots SubcategoryPage default 1`] = `
<DocumentFragment>
  <div
    class="box-adaptive__BoxStyled-sc-g2wy39-0 jBnAwc"
  >
    <div
      class="page-section__SectionStyled-sc-rosswz-0 byJLpF"
    >
      <div
        class="box-adaptive__BoxStyled-sc-g2wy39-0 knChur articles-list__ArticlesStyled-sc-1ql5vrh-0"
        data-test="articles-list-on-page-Content"
      >
        <div
          class="box-adaptive__BoxStyled-sc-g2wy39-0 djPBEv"
          data-test="container"
        >
          <div
            class="box-adaptive__BoxStyled-sc-g2wy39-0"
          >
            <h1
              class="text-content__TextStyled-sc-1stkrsz-0 juAimp articles-list__TextContentStyled-sc-1ql5vrh-1 icHrtT"
              data-test="text-content"
            >
              <span
                class="text-content__TextStyled-sc-1stkrsz-0 hLzCBq"
                data-test="category-title"
              >
                Content
              </span>
            </h1>
          </div>
          <div
            class="box-adaptive__BoxStyled-sc-g2wy39-0 dSDZBu"
            data-test="columns"
          />
        </div>
      </div>
    </div>
    <div
      class="page-section__SectionStyled-sc-rosswz-0 byJLpF"
    >
      <div
        class="box-adaptive__BoxStyled-sc-g2wy39-0 clUAxc block__BlockStyled-sc-h0mnyi-0 gMzVTU"
        data-test="subscription-block"
      >
        <div
          class="box-adaptive__BoxStyled-sc-g2wy39-0 djPBEv"
          data-test="container"
        >
          <div
            class="box-adaptive__BoxStyled-sc-g2wy39-0 hoIJKK"
          >
            <div
              class="box-adaptive__BoxStyled-sc-g2wy39-0 hrMNQR"
              data-test="subscription-content"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 gtIEcS"
              >
                <form
                  data-test="marketo-form"
                  id="mktoForm_1066"
                  style="display: none;"
                />
                <form
                  class="subscription-form__Form-sc-1yspurb-0 laUxsH"
                >
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 cmVuBs"
                  >
                    <div
                      class="text-content__TextStyled-sc-1stkrsz-0 eIieFX"
                      data-test="subscription-title"
                    >
                      Give me the latest news!
                    </div>
                    <div
                      class="text-content__TextStyled-sc-1stkrsz-0 hxphSF"
                      data-test="subscription-text"
                    >
                      Want to know which websites saw the most traffic growth in your industry? Not sure why your SEO strategy doesn’t work? Or simply looking for SEO tips? Subscribe to our newsletter to receive updates on the content you care about.
                    </div>
                  </div>
                  <span
                    class="field-container__Container-sc-qv1os7-0 trAty"
                    data-test="field-container"
                  >
                    <span>
                      <div
                        class="___SInput_ig413_gg_ _size_xl_ig413_gg_ _state_normal_ig413_gg_"
                        data-test="subscription-input"
                        data-ui-name="Input"
                        style="margin-bottom: 12px;"
                      >
                        <input
                          class="___SValue_ig413_gg_ _size_xl_ig413_gg_"
                          data-ui-name="Input.Value"
                          name="email"
                          placeholder="Your email"
                          value=""
                        />
                        <div
                          class="___SOutline_ig413_gg_"
                        />
                      </div>
                    </span>
                  </span>
                  <span
                    class="base-button__BaseButton-sc-1hqbwl-0 internal-button__InternalButton-sc-6tgax4-0 dGmWiH btMXHM"
                    data-test="button-secondary"
                  >
                    <button
                      class="___SButton_rkzvx_gg_ _size_xl_rkzvx_gg_ _theme_primary-info_rkzvx_gg_"
                      data-ui-name="Button"
                      style="margin-top: 12px; width: 100%;"
                      tabindex="0"
                      type="submit"
                    >
                      <span
                        class="___SInner_rkzvx_gg_"
                        data-ui-name="Box"
                      >
                        <span
                          class="_size_xl_rkzvx_gg_ ___SText_rkzvx_gg_"
                          data-ui-name="Button.Text"
                        >
                          Subscribe
                        </span>
                      </span>
                    </button>
                  </span>
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 subscription-form__TipStyled-sc-1yspurb-1 eqlqwJ"
                  >
                    <div
                      class="text-content__TextStyled-sc-1stkrsz-0 gMxUeL"
                      data-test="subscription-tip"
                    >
                      <span
                        class="text-content__TextStyled-sc-1stkrsz-0 bUJWgm"
                        data-test="text-content"
                      >
                        By clicking “Subscribe” you agree to example 
                        <a
                          data-test="subscribe-link-privacy"
                          href="https://www.example.com/company/legal/privacy-policy/"
                          target="_blank"
                        >
                          <span
                            class="text-content__TextStyled-sc-1stkrsz-0 bUJWgm block-with-text__PrivacyLink-sc-gifs8j-0 XoebA"
                            data-test="text-content"
                          >
                            Privacy Policy
                          </span>
                        </a>
                         and consent to example using your contact data for newsletter purposes
                      </span>
                    </div>
                  </div>
                </form>
              </div>
            </div>
            <div
              class="box-adaptive__BoxStyled-sc-g2wy39-0 dLzssZ block__ImageStyled-sc-h0mnyi-1 glxix"
              data-test="subscription-image"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 ipNLfw"
              >
                <img
                  alt="Newsletter example banner"
                  class="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
                  data-src="https://static.example.com/blog-next-static/banners/newsletter.png"
                  data-test="banner-subscribe-png"
                  height="360"
                  width="367"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="page-section__SectionStyled-sc-rosswz-0 kOYoAQ"
    >
      <div
        class="box-adaptive__BoxStyled-sc-g2wy39-0 knChur articles-list__ArticlesStyled-sc-1ql5vrh-0"
        data-test="articles-list-bottom-page"
      >
        <div
          class="box-adaptive__BoxStyled-sc-g2wy39-0 djPBEv"
          data-test="container"
        >
          <div
            class="box-adaptive__BoxStyled-sc-g2wy39-0"
          />
          <div
            class="box-adaptive__BoxStyled-sc-g2wy39-0 dSDZBu"
            data-test="columns"
          >
            <div
              class="column__ColumnStyled-sc-xt4v4z-0 bAzzVW"
              data-test="post-card-small"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 uifxx"
                data-test="post-card-item-en"
              >
                <div
                  class="box-adaptive__BoxStyled-sc-g2wy39-0 VYInM post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
                  data-test="image"
                >
                  <a
                    data-test="image-link"
                    href="/white-hat-seo"
                  >
                    <img
                      alt="Article Image"
                      class="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
                      data-src="https://storage.googleapis.com/blog/testing/article-list/1.jpg"
                      height="190"
                      width="158"
                    />
                  </a>
                </div>
                <article
                  class="post-card-content__ContentStyled-sc-gda4z3-1 cBpLBb"
                  data-test="content-en"
                >
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 jDLZGW"
                  >
                    <div
                      class="post-card-category__DetailsStyled-sc-yr5537-2 bhQdVr"
                    >
                      <p
                        class="text-content__TextStyled-sc-1stkrsz-0 kszZkd post-card-category__CategoryStyled-sc-yr5537-3 lnUjcU"
                        data-test="category"
                      >
                        <a
                          href="/blog/category/advertising"
                          target="_blank"
                        >
                          Advertising
                        </a>
                      </p>
                      <div
                        class="box-adaptive__BoxStyled-sc-g2wy39-0 eZa-dHq post-card-category__TimeToReadStyled-sc-yr5537-0 fZVgVc"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 eljVVb"
                          data-test="time-to-read"
                        >
                          15 min read
                        </span>
                      </div>
                    </div>
                  </div>
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 csiPyy"
                  >
                    <h3
                      class="text-content__TextStyled-sc-1stkrsz-0 hHiLZj"
                      data-test="text-content"
                      style="word-break: break-word;"
                    >
                      <a
                        class="post-card-content__HeadingLinkStyled-sc-gda4z3-0 iOoMty"
                        data-test="title"
                        href="/white-hat-seo"
                      >
                        The Ultimate Guide to Google Search Operators and Google Search Commands
                      </a>
                    </h3>
                    <span
                      class="text-content__TextStyled-sc-1stkrsz-0 kdmpti post-card-content__CaptionStyled-sc-gda4z3-3 iLbRjP"
                      data-test="description"
                    >
                      In this guide, we’ll delve deep into the basics of search operators and search commands, helping you understand how to wield these powerful tools effectively and move on to advanced commands and operators.
                    </span>
                    <div
                      class="box-adaptive__BoxStyled-sc-g2wy39-0 cGxrni post-card-content__MetaStyled-sc-gda4z3-4 cgjjhY"
                      data-test="meta"
                    >
                      <a
                        data-test="author-name"
                        href="/user/olga-andrienko"
                        target="_blank"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 izRvWD post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                          data-test="text-content"
                        >
                          Olga Andrienko
                        </span>
                      </a>
                      <span
                        class="text-content__TextStyled-sc-1stkrsz-0 kHbJxs post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                        data-test="date"
                      >
                        Aug 20, 2020
                      </span>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              class="column__ColumnStyled-sc-xt4v4z-0 bAzzVW"
              data-test="post-card-small"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 uifxx"
                data-test="post-card-item-en"
              >
                <div
                  class="box-adaptive__BoxStyled-sc-g2wy39-0 VYInM post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
                  data-test="image"
                >
                  <a
                    data-test="image-link"
                    href="/how-to-rank-a-youtube-video"
                  >
                    <img
                      alt="Article Image"
                      class="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
                      data-src="https://storage.googleapis.com/blog/testing/article-list/3.jpg"
                      height="190"
                      width="158"
                    />
                  </a>
                </div>
                <article
                  class="post-card-content__ContentStyled-sc-gda4z3-1 cBpLBb"
                  data-test="content-en"
                >
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 jDLZGW"
                  >
                    <div
                      class="post-card-category__DetailsStyled-sc-yr5537-2 bhQdVr"
                    >
                      <div
                        class="box-adaptive__BoxStyled-sc-g2wy39-0 dzytji post-card-category__IconStyled-sc-yr5537-1 bLpcYW"
                      >
                        ogonek
                      </div>
                      <p
                        class="text-content__TextStyled-sc-1stkrsz-0 kszZkd post-card-category__CategoryStyled-sc-yr5537-3 lnUjcU"
                        data-test="category"
                      >
                        <a
                          href="/blog/category/seo"
                          target="_blank"
                        >
                          Seo
                        </a>
                      </p>
                      <div
                        class="box-adaptive__BoxStyled-sc-g2wy39-0 eZa-dHq post-card-category__TimeToReadStyled-sc-yr5537-0 fZVgVc"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 eljVVb"
                          data-test="time-to-read"
                        >
                          16 min read
                        </span>
                      </div>
                    </div>
                  </div>
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 csiPyy"
                  >
                    <h3
                      class="text-content__TextStyled-sc-1stkrsz-0 hHiLZj"
                      data-test="text-content"
                      style="word-break: break-word;"
                    >
                      <a
                        class="post-card-content__HeadingLinkStyled-sc-gda4z3-0 iOoMty"
                        data-test="title"
                        href="/how-to-rank-a-youtube-video"
                      >
                        Some article text with captions and tool words and third party keys and other stuff
                      </a>
                    </h3>
                    <span
                      class="text-content__TextStyled-sc-1stkrsz-0 kdmpti post-card-content__CaptionStyled-sc-gda4z3-3 iLbRjP"
                      data-test="description"
                    >
                      In this guide, we’ll delve deep into the basics of search operators and search commands, helping you understand how to wield these powerful tools effectively and move on to advanced commands and operators.
                    </span>
                    <div
                      class="box-adaptive__BoxStyled-sc-g2wy39-0 cGxrni post-card-content__MetaStyled-sc-gda4z3-4 cgjjhY"
                      data-test="meta"
                    >
                      <a
                        data-test="author-name"
                        href="/user/ivan-petrov"
                        target="_blank"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 izRvWD post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                          data-test="text-content"
                        >
                          Ivan Petrov
                        </span>
                      </a>
                      <span
                        class="text-content__TextStyled-sc-1stkrsz-0 kHbJxs post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                        data-test="date"
                      >
                        Aug 20, 2020
                      </span>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              class="column__ColumnStyled-sc-xt4v4z-0 bAzzVW"
              data-test="post-card-small"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 uifxx"
                data-test="post-card-item-en"
              >
                <div
                  class="box-adaptive__BoxStyled-sc-g2wy39-0 VYInM post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
                  data-test="image"
                >
                  <a
                    data-test="image-link"
                    href="/improve-seo"
                  >
                    <img
                      alt="Article Image"
                      class="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
                      data-src="https://storage.googleapis.com/blog/testing/article-list/4.jpg"
                      height="190"
                      width="158"
                    />
                  </a>
                </div>
                <article
                  class="post-card-content__ContentStyled-sc-gda4z3-1 cBpLBb"
                  data-test="content-en"
                >
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 jDLZGW"
                  >
                    <div
                      class="post-card-category__DetailsStyled-sc-yr5537-2 bhQdVr"
                    >
                      <p
                        class="text-content__TextStyled-sc-1stkrsz-0 kszZkd post-card-category__CategoryStyled-sc-yr5537-3 lnUjcU"
                        data-test="category"
                      >
                        <a
                          href="/blog/category/seo"
                          target="_blank"
                        >
                          Seo
                        </a>
                      </p>
                      <div
                        class="box-adaptive__BoxStyled-sc-g2wy39-0 eZa-dHq post-card-category__TimeToReadStyled-sc-yr5537-0 fZVgVc"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 eljVVb"
                          data-test="time-to-read"
                        >
                          2 min read
                        </span>
                      </div>
                    </div>
                  </div>
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 csiPyy"
                  >
                    <h3
                      class="text-content__TextStyled-sc-1stkrsz-0 hHiLZj"
                      data-test="text-content"
                      style="word-break: break-word;"
                    >
                      <a
                        class="post-card-content__HeadingLinkStyled-sc-gda4z3-0 iOoMty"
                        data-test="title"
                        href="/improve-seo"
                      >
                        A article text with captions and tool words and third party keys and other stuff
                      </a>
                    </h3>
                    <span
                      class="text-content__TextStyled-sc-1stkrsz-0 kdmpti post-card-content__CaptionStyled-sc-gda4z3-3 iLbRjP"
                      data-test="description"
                    >
                      In this guide, we’ll delve deep into the basics of search operators and search commands, helping you understand how to wield these powerful tools effectively and move on to advanced commands and operators.
                    </span>
                    <div
                      class="box-adaptive__BoxStyled-sc-g2wy39-0 cGxrni post-card-content__MetaStyled-sc-gda4z3-4 cgjjhY"
                      data-test="meta"
                    >
                      <a
                        data-test="author-name"
                        href="/user/olga-andrienko"
                        target="_blank"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 izRvWD post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                          data-test="text-content"
                        >
                          Olga Andrienko
                        </span>
                      </a>
                      <span
                        class="text-content__TextStyled-sc-1stkrsz-0 kHbJxs post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                        data-test="date"
                      >
                        Aug 20, 2020
                      </span>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              class="column__ColumnStyled-sc-xt4v4z-0 bAzzVW"
              data-test="post-card-small"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 uifxx"
                data-test="post-card-item-en"
              >
                <div
                  class="box-adaptive__BoxStyled-sc-g2wy39-0 VYInM post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
                  data-test="image"
                >
                  <a
                    data-test="image-link"
                    href="/improve-seo"
                  >
                    <img
                      alt="Article Image"
                      class="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
                      data-src="https://storage.googleapis.com/blog/testing/article-list/2.jpg"
                      height="190"
                      width="158"
                    />
                  </a>
                </div>
                <article
                  class="post-card-content__ContentStyled-sc-gda4z3-1 cBpLBb"
                  data-test="content-en"
                >
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 jDLZGW"
                  >
                    <div
                      class="post-card-category__DetailsStyled-sc-yr5537-2 bhQdVr"
                    >
                      <p
                        class="text-content__TextStyled-sc-1stkrsz-0 kszZkd post-card-category__CategoryStyled-sc-yr5537-3 lnUjcU"
                        data-test="category"
                      >
                        <a
                          href="/blog/category/social-media"
                          target="_blank"
                        >
                          Social Media
                        </a>
                      </p>
                      <div
                        class="box-adaptive__BoxStyled-sc-g2wy39-0 eZa-dHq post-card-category__TimeToReadStyled-sc-yr5537-0 fZVgVc"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 eljVVb"
                          data-test="time-to-read"
                        >
                          44 min read
                        </span>
                      </div>
                    </div>
                  </div>
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 csiPyy"
                  >
                    <h3
                      class="text-content__TextStyled-sc-1stkrsz-0 hHiLZj"
                      data-test="text-content"
                      style="word-break: break-word;"
                    >
                      <a
                        class="post-card-content__HeadingLinkStyled-sc-gda4z3-0 iOoMty"
                        data-test="title"
                        href="/improve-seo"
                      >
                        A article text with captions and tool words and third party keys and other stuff
                      </a>
                    </h3>
                    <span
                      class="text-content__TextStyled-sc-1stkrsz-0 kdmpti post-card-content__CaptionStyled-sc-gda4z3-3 iLbRjP"
                      data-test="description"
                    >
                      In this guide, we’ll delve deep into the basics of search operators and search commands, helping you understand how to wield these powerful tools effectively and move on to advanced commands and operators.
                    </span>
                    <div
                      class="box-adaptive__BoxStyled-sc-g2wy39-0 cGxrni post-card-content__MetaStyled-sc-gda4z3-4 cgjjhY"
                      data-test="meta"
                    >
                      <a
                        data-test="author-name"
                        href="/user/ivan-petrov"
                        target="_blank"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 izRvWD post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                          data-test="text-content"
                        >
                          Ivan Petrov
                        </span>
                      </a>
                      <span
                        class="text-content__TextStyled-sc-1stkrsz-0 kHbJxs post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                        data-test="date"
                      >
                        Aug 20, 2020
                      </span>
                    </div>
                  </div>
                </article>
              </div>
            </div>
            <div
              class="column__ColumnStyled-sc-xt4v4z-0 bAzzVW"
              data-test="post-card-small"
            >
              <div
                class="box-adaptive__BoxStyled-sc-g2wy39-0 uifxx"
                data-test="post-card-item-en"
              >
                <div
                  class="box-adaptive__BoxStyled-sc-g2wy39-0 VYInM post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
                  data-test="image"
                >
                  <a
                    data-test="image-link"
                    href="/improve-seo"
                  >
                    <img
                      alt="Article Image"
                      class="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
                      data-src="https://storage.googleapis.com/blog/testing/article-list/3.jpg"
                      height="190"
                      width="158"
                    />
                  </a>
                </div>
                <article
                  class="post-card-content__ContentStyled-sc-gda4z3-1 cBpLBb"
                  data-test="content-en"
                >
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 jDLZGW"
                  >
                    <div
                      class="post-card-category__DetailsStyled-sc-yr5537-2 bhQdVr"
                    >
                      <p
                        class="text-content__TextStyled-sc-1stkrsz-0 kszZkd post-card-category__CategoryStyled-sc-yr5537-3 lnUjcU"
                        data-test="category"
                      >
                        <a
                          href="/blog/category/market-research"
                          target="_blank"
                        >
                          Market Research
                        </a>
                      </p>
                      <div
                        class="box-adaptive__BoxStyled-sc-g2wy39-0 eZa-dHq post-card-category__TimeToReadStyled-sc-yr5537-0 fZVgVc"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 eljVVb"
                          data-test="time-to-read"
                        >
                          5 min read
                        </span>
                      </div>
                    </div>
                  </div>
                  <div
                    class="box-adaptive__BoxStyled-sc-g2wy39-0 csiPyy"
                  >
                    <h3
                      class="text-content__TextStyled-sc-1stkrsz-0 hHiLZj"
                      data-test="text-content"
                      style="word-break: break-word;"
                    >
                      <a
                        class="post-card-content__HeadingLinkStyled-sc-gda4z3-0 iOoMty"
                        data-test="title"
                        href="/improve-seo"
                      >
                        A article text with captions and tool words and third party keys and other stuff
                      </a>
                    </h3>
                    <span
                      class="text-content__TextStyled-sc-1stkrsz-0 kdmpti post-card-content__CaptionStyled-sc-gda4z3-3 iLbRjP"
                      data-test="description"
                    >
                      In this guide, we’ll delve deep into the basics of search operators and search commands, helping you understand how to wield these powerful tools effectively and move on to advanced commands and operators.
                    </span>
                    <div
                      class="box-adaptive__BoxStyled-sc-g2wy39-0 cGxrni post-card-content__MetaStyled-sc-gda4z3-4 cgjjhY"
                      data-test="meta"
                    >
                      <a
                        data-test="author-name"
                        href="/user/olga-andrienko"
                        target="_blank"
                      >
                        <span
                          class="text-content__TextStyled-sc-1stkrsz-0 izRvWD post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                          data-test="text-content"
                        >
                          Olga Andrienko
                        </span>
                      </a>
                      <span
                        class="text-content__TextStyled-sc-1stkrsz-0 kHbJxs post-card-content__NoWrapStyled-sc-gda4z3-2 jqrfVT"
                        data-test="date"
                      >
                        Aug 20, 2020
                      </span>
                    </div>
                  </div>
                </article>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="box-adaptive__BoxStyled-sc-g2wy39-0 djPBEv"
        data-test="container"
      >
        <div
          class="box-adaptive__BoxStyled-sc-g2wy39-0 dAuyOH"
        >
          <span
            class="text-content__TextStyled-sc-1stkrsz-0 gNQKEc"
            data-test="text-content"
          >
            More about Seo
          </span>
          <div
            class="box-adaptive__BoxStyled-sc-g2wy39-0 dkxOHA"
          >
            <span
              class="text-content__TextStyled-sc-1stkrsz-0 gnSSuA tag-control__TagControlStyled-sc-5ljs9a-0 fuGmRv"
              data-test="tag-control"
            >
              <a
                href="/blog/category/seo/ecommerce"
              >
                eCommerce
              </a>
            </span>
            <span
              class="text-content__TextStyled-sc-1stkrsz-0 gnSSuA tag-control__TagControlStyled-sc-5ljs9a-0 fuGmRv"
              data-test="tag-control"
            >
              <a
                href="/blog/category/seo/market-research"
              >
                Market Research
              </a>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
